xbmc/network/websocket/WebSocket.h

```diff
@@ -121,7 +121,7 @@ class CWebSocket
   virtual const CWebSocketMessage* Handle(const char* &buffer, size_t &length, bool &send);
   virtual const CWebSocketMessage* Send(WebSocketFrameOpcode opcode, const char* data = NULL, uint32_t length = 0);
   virtual const CWebSocketFrame* Ping(const char* data = NULL) const = 0;
-  virtual const CWebSocketFrame* Pong(const char* data = NULL) const = 0;
+  virtual const CWebSocketFrame* Pong(const char* data = NULL, uint32_t length = 0) const = 0;
```

Comment:
Is the default value of 0 really needed?

Reply:
It is, if we keep the data optional.
There is only one implementation, so it's trivial to remove them both.
It's a matter of taste, application data is optional for a pong, so semantically I think the defaults would be fine.
Changed it anyway, I'll keep whatever is preferred.